src/components/nav-bar/DesktopContent.jsx

```diff
@@ -13,10 +14,11 @@ const DesktopContent = ({ desktopLeftContent, desktopRightContent }) => {
             className="button-dark"
             onClick={(e) => {
               e.preventDefault();
-              window.location.href = '/';
             }}
           >
-            <HubblrImage className="w-20 h-auto" />
+            <Link to="/">
```

Comment:
Link im Button ist eher unsauber. Denke du hast hier 2 Alternativen: Entweder du benutzt im `onClick` das `navigate` von Gatsby (https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-link/#how-to-use-the-navigate-helper-function) oder du benutzt den Link und stylest den wie den AppButton (der hat nur "button", wäre also relativ simpel hier).

Reply:
Guter Punkt. Habe es jetzt zum Link mit Button Style gemacht, da er mir bei navigate die Fehlermeldung gegeben hat, er findet das Object() nicht, wenn ich zu "/" linken wollte...
Allerdings ist es generell so, dass auf der Main Page nicht wieder zum Anfang gescrollt wird, wenn man klickt - geht das auch irgendwie? habe überall nur window.scrollTo(0,0) gefunden?

Reply:
Also window.scrollTo(0, 0) ist schon das, was du da ausführen müsstest dann. Am besten wäre glaube ich doch über navigate das zu machen weil das automatisch scrollt (oder wenn nicht mit einer Funktion die den scroll im callback von navigate ausführt). Importierst du das denn korrekt? Das ist `import { navigate } from '@reach/router'.`

Sonst kannst du den scroll auch in einem Hook/useLayoutEffect z.B. im Layout machen.

Reply:
Hab es gerade wieder mit dem Navigate versucht, aber leider funktioniert das auf der MainPage nicht bzw. nur einmal und dann nicht mehr (auch nicht mit window.scrollTo(0,0)...) - eine Idee wieso?

Reply:
Done.